dolphinscheduler-dao/pom.xml

```diff
@@ -106,10 +106,6 @@
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.alibaba</groupId>
```

Comment:
Remove the license in all LICENSE file and known-dependencies.txt

Reply:
At present, dataX-task strongly relies on Druid for parsing. It is recommended to switch the metadata to provide interface parsing and then remove it

Reply:
Got it